atlas/query_node_API_sketch.graphql

```diff
@@ -0,0 +1,196 @@
+
+enum Language {
+  Chinese
+  English
+  Arabic
+  Portugese
+  French
+  # ...
+}
+
+type Channel {
+
+  id: ID!
+
+  # Id of underlying entity.
+  entityID: BigInteger!
+
+  owner: Member!
+
+  handle: String!
+
+  description: String!
+
+  coverPhotoURL: String!
+
+  avatarPhotoURL: String!
+
+  isPublic: Bool!
+
+  isCurated: Boolean!
+
+  language: Language
+
+  videos: [Video!]
```

Comment:
Shouldn't those by paginated?

If so, I'd expect a different return type, more specifically what I've mentioned in

Reply:
You mean  `videos`? if so, no, its just a collection of all videos in the channel. If you want a paginated list of all videos _in a given channel_, you have to do a video query, and filter on the channel membership property, and then add pagination.

Reply:
Okay, thanks